src/wiki/views/article.py

```diff
@@ -565,7 +564,7 @@ def get_context_data(self, **kwargs):
         if 'form' not in kwargs:
             kwargs['form'] = self.get_form()
         kwargs['purge_form'] = kwargs.pop('form', None)
-        return super(Delete, self).get_context_data(**kwargs)
+        return super().get_context_data(**kwargs)
```

Comment:
This one wasn't calling `Delete.get_context_data` but the parent's `get_context_data`!

Reply:
Can't really tell if it's intentional or not though :)

Reply:
Just to be safe, I'll switch back to the old style with `Delete`. The mismatch between `Derived` and `Parent` as the first argument to `super` is fairly weird to me, though.